language/stdlib/modules/ValidatorConfig.move

```diff
@@ -275,16 +278,18 @@ module ValidatorConfig {
     }
 
     /// Get operator's account
-    /// Aborts if there is no ValidatorConfig resource, if its operator_account is
-    /// empty, returns the input
+    /// Aborts if there is no ValidatorConfig resource or
+    /// if the operator_account is unset
     public fun get_operator(addr: address): address acquires ValidatorConfig {
         assert(exists<ValidatorConfig>(addr), Errors::not_published(EVALIDATOR_CONFIG));
         let t_ref = borrow_global<ValidatorConfig>(addr);
-        *Option::borrow_with_default(&t_ref.operator_account, &addr)
+        assert(Option::is_some(&t_ref.operator_account), Errors::invalid_argument(EVALIDATOR_CONFIG));
+        *Option::borrow(&t_ref.operator_account)
     }
 
     spec fun get_operator {
         pragma opaque;
+        pragma verify_duration_estimate = 100;
```

Comment:
I think this now needs an `aborts_if` for case where `operator_account` is None. Actually, I'm surprised the prover tests are passing without this. Do you know why @wrwg?